### PR TITLE
Fixed all Grammatical typo's in README.md

### PR DESCRIPTION
**Pull request by prakashAditya639**
1 - In the sentence, "We empower developers to change how they write applications, support operators in rethinking how they run modern infrastructure," "rethinking" should be spelled as "re-think."

2 - In the "MeshMap" section, "deploy designs" should be "deploys designs" for proper verb agreement.

3 - In the "Meshery Operator" section, "Meshery operatorMeshery Operator deploys and manages MeshSync" lacks a space between "Meshery operator" and "Meshery Operator." It should be "Meshery operator - Meshery Operator deploys and manages MeshSync."

4 - In the "Community and Contributions" section, "whether your opening an issue or a pull requests" should be "whether you're opening an issue or a pull request" for proper grammar. Also, "pull requests" should be "pull request" if referring to a single action.

5 - In the "Community and Contributions" section, "Join us in the discussion forum and on Slack" should be "Join us in the discussion forum and on Slack" without an extra space before "Slack."

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 23a3d4d9fe392cda6bff9af2dc9eed088d9ee6cb at: https://651f993d52eb965ffccbe31d--layer5.netlify.app

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 # About Layer5
 
-[Layer5](https://layer5.io)'s cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in rethinking how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
+[Layer5](https://layer5.io)'s cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in re-think how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
```
Comment (leecalcote) [inline README.md:26]: Thanks for these corrections, @prakashAditya639. 👍 In this case, "rethinking" is grammatically correct. We'll want to leave this one as-is.

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@
 <a href="https://layer5.io/meshmap">
   <img src="src/assets/images/meshmap/meshmap-color-icon.svg" alt="MeshMap logo" width="125px" style="margin:10px;" align="left" />
 </a>
-<a href="https://layer5.io/meshmap">MeshMap</a> is an end-to-end management platform, here to help teams understand problems, deploy designs, apply patterns, manage and operate your deployments and services in real-time.
+<a href="https://layer5.io/meshmap">MeshMap</a> is an end-to-end management platform, here to help teams understand problems, deploys designs, apply patterns, manage and operate your deployments and services in real-time.
```
Comment (leecalcote) [inline README.md:46]: Incorrect. Please revert.

File: README.md [hunk 3]
```diff
@@ -98,12 +98,12 @@ Image Hub supports Envoy-based data planes. It is compatible with and can be dep
 
 
 <p style="clear:both;">
- <h2><a href="https://github.com/meshery/meshery-operator">Meshery Operator</a></h2>
+ <h2><a href="https://github.com/meshery/meshery-operator">Meshery Operator: </a></h2>
```
Comment (leecalcote) [inline README.md:101]: No thanks. Let's stay sans colon.

File: README.md [hunk 4]
```diff
@@ -98,12 +98,12 @@ Image Hub supports Envoy-based data planes. It is compatible with and can be dep
...
  <a href="https://meshery.io">
    <img src=".github/assets/images/meshery-operator/meshery-operator-dark.svg"
  style="float:left;margin:10px;" width="125px" alt="Meshery operator" align="left" />
 </a>
- <a href="https://github.com/layer5io/meshery-operator">Meshery Operator</a> deploys and manages MeshSync.
+ <a href="https://github.com/layer5io/meshery-operator"> Meshery Operator</a> deploys and manages MeshSync.
```
Comment (leecalcote) [inline README.md:106]: Good.

File: README.md [hunk 5]
```diff
@@ -159,10 +159,10 @@ alt="Meshery Catalog" align="left" />
...
+<p> We warmly welcome all contributors! Our projects are community-built and each welcomes open collaboration. As you get started, please review this project's <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">contributing guidelines</a>. Whether you are a user or code contributor and whether you're opening an <a href="/../../issues">issue</a> or a <a href="/../../pulls">pull request</a>, know that any form of your engagement is considered contribution and is appreciated. Contributors are expected to adhere to the <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Code of Conduct</a>.
```
Comment (leecalcote) [inline README.md:162]: Good eye.

File: README.md [hunk 6]
```diff
@@ -159,10 +159,10 @@ alt="Meshery Catalog" align="left" />
...
 </p>
 <p>
-  Join us in the <a href="https://discuss.layer5.io">discussion forum</a> and on <a href="https://slack.layer5.io"><img src=".github/assets/images/slack.webp" height="16px" align="bottom" /> Slack</a> to learn more about Layer5 and its community! Make sure you see the <a href="https://layer5.io/community/newcomers">newcomer's guide</a> for a tour of resources available to you.
+  Join us in the <a href="https://discuss.layer5.io">discussion forum</a> and on <a href="https://slack.layer5.io"><img src=".github/assets/images/slack.webp" height="16px" align="bottom" />Slack</a> to learn more about Layer5 and its community! Make sure you see the <a href="https://layer5.io/community/newcomers">newcomer's guide</a> for a tour of resources available to you.
```
Comment (leecalcote) [inline README.md:165]: Please revert given that this whitespace is for purposes of maintaining distance between the image and the word.

Screenshot 2023-10-07 at 1 05 48 PM

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by prakashAditya639** [comment]
Reverted Back all the changes you requested for.....

**Review by leecalcote — APPROVED** [review]
Thanks!

**Pull request merged**